modules/default/compliments/compliments.js

```diff
@@ -39,7 +41,8 @@ Module.register("compliments", {
 		afternoonStartTime: 12,
 		afternoonEndTime: 17,
 		random: true,
-		mockDate: null
+		mockDate: null,
+    advice: false
```

Comment:
please use the same intendation here as in the lines above. makes the code more readable.